src/helper/response-handler.js

```diff
@@ -42,7 +42,10 @@ function wrapCallback(cb, options) {
       if (err.err) {
         err = err.err;
       }
-
+      if (options.forceLegacyError) {
+        errObj.error = err.error;
```

Comment:
This means that it's possible for `errObj.code` !== `errObj.error` ... is that intended? Maybe this should fall after where `err.code` and `errObj.description` are set and set them as equal?

Reply:
makes sense. pushed a change